From 48feadc5fb495f311230cd3454cdf1c22519a794 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Wed, 13 Feb 2019 15:04:59 +0200
Subject: [PATCH 03/49] ucm-imx8m-mini: enable status LED

Use the status LED feature to turn on the status LED.

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 configs/ucm-imx8m-mini_defconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/configs/ucm-imx8m-mini_defconfig b/configs/ucm-imx8m-mini_defconfig
index 2212e76acb..36cfe1b2e1 100644
--- a/configs/ucm-imx8m-mini_defconfig
+++ b/configs/ucm-imx8m-mini_defconfig
@@ -24,6 +24,7 @@ CONFIG_CMD_MEMTEST=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_CACHE=y
+# CONFIG_CMD_LED is not set
 CONFIG_CMD_REGULATOR=y
 CONFIG_CMD_FS_GENERIC=y
 CONFIG_EFI_PARTITION=y
@@ -32,6 +33,11 @@ CONFIG_ENV_IS_IN_MMC=y
 CONFIG_DM_GPIO=y
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_MXC=y
+CONFIG_LED=y
+CONFIG_LED_STATUS=y
+CONFIG_LED_STATUS_GPIO=y
+CONFIG_LED_STATUS0=y
+CONFIG_LED_STATUS_BIT=12
 CONFIG_DM_MMC=y
 CONFIG_DM_SPI_FLASH=y
 CONFIG_SPI_FLASH=y
-- 
2.11.0

